### ISSUE TITLE
Add StrykerJS test

### ISSUE THREAD

**Issue by nicojs** [vitest-dev/vitest-ecosystem-ci#11]
Hi there 🙋‍♂️

I've found some issues in vitest, and now and again, I also find some regressions. @AriPerkkio pointed me here to contribute a test, maybe.

https://github.com/vitest-dev/vitest/issues/4713#issuecomment-1848401077

However, StrykerJS is different from your run-of-the-mil repo since Stryker is a framework in which vitest is used via the [@stryker-mutator/vitest-runner](https://stryker-mutator.io/docs/stryker-js/vitest-runner/) plugin. 

The simplest way to add a test is to rebrand some of our [e2e tests](https://github.com/stryker-mutator/stryker-js/tree/master/e2e). Either create separate repos or add them directly here (they're relatively small projects). A test would then run StrykerJS via the command line and snapshot test the JSON result (which contains mutation score and other metrics).

Another question I have is how to install vitest inside these e2e test projects. Inside StrykerJS, we use [install-local](https://www.npmjs.com/package/install-local), a small package I'm maintaining. It can install a local package (not symlinking, but by running `npm pack` and installing that package), but I'm not sure if that is desirable or maintainable here since I see `pnpm` being used here.

**Comment by sheremet-va** [comment]
How do you run Vitest tests now in the StrykerJS CI? Is it public?

**Comment by sheremet-va** [comment]
On the same note, are you present in Discord's ecosystem channel? We try to notify libraries that have Vitest integrations about changes that might potentially break things. If you are no there, just message me and I will send you the invite.

**Comment by nicojs** [comment]
Yes we run Stryker on some vitest projects as part of our e2e test suite here: https://github.com/stryker-mutator/stryker-js/tree/master/e2e%2Ftest

I will join the discord. Thanks for the advice. Do you still want me to move one of those projects here? Or maybe update the vitest version on our end to the beta version before major releases is a better way to go?

**Comment by sheremet-va** [comment]
> Do you still want me to move one of those projects here? Or maybe update the vitest version on our end to the beta version before major releases is a better way to go?

This repository doesn't have any projects, you need to add a configuration that will install another repository (yours) and run those tests in our CI. If you add a command like `test:vitest` and specify it here, it should be enough. You can see an example here: https://github.com/vitest-dev/vitest-ecosystem-ci/blob/main/tests/nuxt.ts

**Comment by sheremet-va** [comment]
> I will join the discord. Thanks for the advice.

I found you on the members list. You should have the "ecosystem" role now with access to the "ecosystem" channel.
